client/app/scripts/superdesk-ingest/module.js

```diff
@@ -59,6 +59,10 @@ define([
         search: {
             label: 'Search provider',
             templateUrl: 'scripts/superdesk-ingest/views/settings/searchConfig.html'
+        },
+        whatsapp: {
```

Comment:
maybe it could be set to constant and not value and then you can do this using in `.config` block and not only `.run` one, but it should work the same